index.md

```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents. It enables the confident and reliable use of the JSON data format.
 
+## Mission
+
+Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules, while enabling and supporting use cases beyond validation.
```

Comment:
I find this more natural without the second comma -- 

```suggestion
Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules while enabling and supporting use cases beyond validation.
```

Content-wise it seems reasonable enough to me though so happy to thumbs up myself if the plan is to address Jason's point above in some other PR later.

Reply:
An editor friend of mine has confirmed that the comma is necessary. Otherwise the meaning sort of changes.